.github/workflows/ci.yml

```diff
@@ -0,0 +1,38 @@
+name: CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    strategy:
+      fail-fast: false
+      matrix:
+        go: [1.13, 1.14]
```

Comment:
Optional: Add 1.15.

Reply:
I will do this in a separate revision shortly, as I wanted to do it all at once with Bigslice and Bigmachine.

Reply:
Okay; since I had to touch things up anyway, I went ahead and added 1.15.